src/plugins/intel_gpu/src/plugin/transformations/kv_cache_fusion.cpp

```diff
@@ -62,9 +64,11 @@ KVCacheFusionMatcher::KVCacheFusionMatcher() {
         if (past_node->get_variable_id() != present_node->get_variable_id())
             return false;
 
-        // TODO: Support conversion internally
-        if (!concat_node || concat_node->get_output_element_type(0) != past_node->get_output_element_type(0))
-            return false;
+        // // TODO: Support conversion internally
```

Comment:
minor: to see if we can remove this check later.

Reply:
ok, thank you.